fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -237,20 +237,15 @@ class GlobalTagThrottlerImpl {
 
 	// For transactions with the provided tag, returns the average cost of all transactions
 	// accross the cluster. The minimum cost is 1.
-	double getAverageTransactionCost(TransactionTag tag) const {
+	double getAverageTransactionCost(TransactionTag tag, TraceEvent& te) const {
 		auto const cost = getCurrentCost(tag);
 		auto const stats = tryGet(tagStatistics, tag);
 		if (!stats.present()) {
 			return CLIENT_KNOBS->TAG_THROTTLING_PAGE_SIZE;
 		}
 		auto const transactionRate = stats.get().getTransactionRate();
-		// FIXME: Disabled due to noisy trace events. Fix the noise and reenabled
-		/*
-		TraceEvent("GlobalTagThrottler_GetAverageTransactionCost")
-		    .detail("Tag", tag)
-		    .detail("TransactionRate", transactionRate)
-		    .detail("Cost", cost);
-		*/
+		te.detail("TransactionRate", transactionRate);
```

Comment:
Optional: I wonder if there's a simple way to include this in `getTargetTps` where the TraceEvent is printed so that it is easy to find what details one should expect to see in this TraceEvent.